quest box not appearing on party page for players using certain languages

We've had several reports of people who are participating in quests not being able to see the quest details on their party page.
I haven't worked out why but I'm seeking more information from the people who've reported it.
Below are the reports we've seen so far in the Report a Bug guild.

---

@-Tyr- (38009d36-5a27-4ef0-a890-e81f4c847425) one to two days ago (this is the first report):

> I currently have no quest display in my party sidebar. We are currently on a quest, but there is no quest boss image, health bar etc.. All there is is the party description and challenges.
>
> ... screenshot of the missing quest details in the party sidebar. [see below for the screenshot]
> This is Chrome web browser on a Windows computer.
>
> Another member of my party (who is also using Chrome on a Windows computer) can see the quest details. We are currently on a quest.

@Transurfer (ffe1c057-537c-454d-873f-c590b162ff2c):

> Something has happened to my party today. We are on the quest but you can't see it on the website. The whole part for the quest is missing.
>
> I've checked the app and the quest is there. On the website still nothing. There is the whole part for the quest missing. It shows Description and Challenges and that's it between Invite and Leave party buttons.

@fernandojtx (a5a8c688-eb0f-40d4-b4ff-b05c99595fa6):

> me and my group were on a quest and it simply disappeared 

@Zelah_Meyer (8ea14435-ab6e-4185-86ac-3d663bf616ba):

> Can confirm that I am also suffering from the quest vanishing on the PC version (I'm in Chrome). It happened yesterday and I had to guess how much damage needed doing to complete the quest. Succeeded in defeating the monster when I logged in this morning (even though it was hidden yesterday). My friend sent out an invite for a new quest today and I could see that but when I got the e-mail to say she'd started the quest, I checked the site and it had vanished again. I can see the monster & my damage on the app, but there's nothing on PC.

@Prairiehamster (a6ca1bdf-510d-4730-b471-4d2a4e3fe719):

> I'm also having an issue with the disappearing quest - Windows 10 PC, using up to date Chrome. I've confirmed we're still fighting the boss - I just did damage to it a few minutes ago and I can see it on my app. But on the pc, nothing.

-----

So far all the reports have either not given platform information or have said that they're on Chrome on Windows. But @-Tyr- has said that a party member who is also using Chrome on Windows CAN see the quest, so it's not necessarily a Chrome problem. This report also means it's not a problem with a specific quest or a specific party.

I can see my party's quest (`atom3`) on Firefox on Ubuntu. I can still see it when I change the quest to be `ghost_stag`, which is the quest that one of the affected players is on.

I've done some comparison of party and quest data between all the reports and can't see anything in common.

I'm asking for console log messages and testing on other browsers.

---

Here's -Tyr-'s screenshot:

![image](https://user-images.githubusercontent.com/1495809/117533782-02fe3680-b032-11eb-993e-26eea9dfd8ce.png)

Hi, this is -Tyr-. I have the JS Console errors:
```
logging.js:30 Unhandled error in Vue.js code.
n.default.config.errorHandler @ logging.js:30
re @ vue.runtime.esm.js:1870
ee @ vue.runtime.esm.js:1839
t._render @ vue.runtime.esm.js:3550
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
(anonymous) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonymous) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (async)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
rr.set @ vue.runtime.esm.js:4631
(anonymous) @ group.vue:592
u @ runtime.js:45
(anonymous) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
Promise.then (async)
r @ asyncToGenerator.js:13
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
fetchGuild @ group.vue:585
(anonymous) @ group.vue:502
u @ runtime.js:45
(anonymous) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
mounted @ group.vue:499
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
insert @ vue.runtime.esm.js:3139
E @ vue.runtime.esm.js:6346
(anonymous) @ vue.runtime.esm.js:6565
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonymous) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (async)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
(anonymous) @ app.vue:400
Promise.then (async)
created @ app.vue:399
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
t._init @ vue.runtime.esm.js:5008
a @ vue.runtime.esm.js:5154
on @ vue.runtime.esm.js:3283
init @ vue.runtime.esm.js:3114
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
(anonymous) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3945
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
t._init @ vue.runtime.esm.js:5018
xr @ vue.runtime.esm.js:5085
56d7 @ main.js:36
c @ bootstrap:89
0 @ app.35ff403e.js:1
c @ bootstrap:89
a @ bootstrap:45
(anonymous) @ bootstrap:267
(anonymous) @ app.35ff403e.js:1
logging.js:31 Error: RangeError: Incorrect locale information provided
    at Number.toLocaleString (<anonymous>)
    at Bh (localizeNumber.js:13)
    at a.ft (questSidebarSection.vue?c281:1)
    at a.t._render (vue.runtime.esm.js:3548)
    at a.r (vue.runtime.esm.js:4066)
    at nr.get (vue.runtime.esm.js:4479)
    at new nr (vue.runtime.esm.js:4468)
    at $n (vue.runtime.esm.js:4073)
    at a.xr.$mount (vue.runtime.esm.js:8415)
    at init (vue.runtime.esm.js:3118)
n.default.config.errorHandler @ logging.js:31
re @ vue.runtime.esm.js:1870
ee @ vue.runtime.esm.js:1839
t._render @ vue.runtime.esm.js:3550
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
(anonymous) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonymous) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (async)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
rr.set @ vue.runtime.esm.js:4631
(anonymous) @ group.vue:592
u @ runtime.js:45
(anonymous) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
Promise.then (async)
r @ asyncToGenerator.js:13
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
fetchGuild @ group.vue:585
(anonymous) @ group.vue:502
u @ runtime.js:45
(anonymous) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
mounted @ group.vue:499
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
insert @ vue.runtime.esm.js:3139
E @ vue.runtime.esm.js:6346
(anonymous) @ vue.runtime.esm.js:6565
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonymous) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (async)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
(anonymous) @ app.vue:400
Promise.then (async)
created @ app.vue:399
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
t._init @ vue.runtime.esm.js:5008
a @ vue.runtime.esm.js:5154
on @ vue.runtime.esm.js:3283
init @ vue.runtime.esm.js:3114
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
(anonymous) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3945
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
t._init @ vue.runtime.esm.js:5018
xr @ vue.runtime.esm.js:5085
56d7 @ main.js:36
c @ bootstrap:89
0 @ app.35ff403e.js:1
c @ bootstrap:89
a @ bootstrap:45
(anonymous) @ bootstrap:267
(anonymous) @ app.35ff403e.js:1
logging.js:32 Component where it occurred: a {_uid: 1043, _isVue: true, $options: {…}, _renderProxy: a, _self: a, …}$attrs: (...)$children: []$createElement: ƒ (e,n,r,i)$el: comment$listeners: (...)$options: {parent: a, _parentVnode: yt, propsData: {…}, _parentListeners: undefined, _renderChildren: undefined, …}$parent: a {_uid: 157, _isVue: true, $options: {…}, _renderProxy: a, _self: a, …}$refs: {}$root: xr {_uid: 3, _isVue: true, $options: {…}, _renderProxy: xr, _self: xr, …}$scopedSlots: {$stable: true, $key: undefined, $hasNormal: false}$slots: {}$store: e {_actions: {…}, getters: {…}, _vm: xr}$vnode: yt {tag: "vue-component-300-quest-sidebar-section", data: {…}, children: undefined, text: undefined, elm: comment, …}acceptedCount: (...)bossHpPercent: (...)canEditQuest: (...)group: (...)icons: (...)isMemberOfPendingQuest: (...)isMemberOfRunningQuest: (...)onActiveQuest: (...)onPendingQuest: (...)openParticipantList: ƒ ()openQuestDetails: ƒ ()openStartQuestModal: ƒ ()questAbort: ƒ ()questAccept: ƒ ()questData: (...)questLeave: ƒ ()questReject: ƒ ()user: (...)userIsOnQuest: (...)_bv__modal: t {_vm: a, _root: xr}_bv__toast: t {_vm: a, _root: xr}_c: ƒ (e,n,r,i)_computedWatchers: {user: nr, userIsOnQuest: nr, onPendingQuest: nr, onActiveQuest: nr, bossHpPercent: nr, …}_data: {__ob__: Et}_directInactive: false_events: {}_hasHookEvent: false_inactive: null_isBeingDestroyed: false_isDestroyed: false_isMounted: true_isVue: true_props: {}_renderProxy: a {_uid: 1043, _isVue: true, $options: {…}, _renderProxy: a, _self: a, …}_routerRoot: xr {_uid: 3, _isVue: true, $options: {…}, _renderProxy: xr, _self: xr, …}_self: a {_uid: 1043, _isVue: true, $options: {…}, _renderProxy: a, _self: a, …}_staticTrees: (5) [empty × 3, yt, yt]_uid: 1043_vnode: yt {tag: undefined, data: undefined, children: undefined, text: "", elm: comment, …}_watcher: nr {vm: a, deep: false, user: false, lazy: false, sync: false, …}_watchers: (11) [nr, nr, nr, nr, nr, nr, nr, nr, nr, nr, nr]$bvModal: (...)$bvToast: (...)$data: (...)$isServer: (...)$props: (...)$route: (...)$router: (...)$ssrContext: (...)get $attrs: ƒ ()set $attrs: ƒ (e)get $listeners: ƒ ()set $listeners: ƒ (e)get icons: ƒ ()set icons: ƒ (t)__proto__: xr
n.default.config.errorHandler @ logging.js:32
re @ vue.runtime.esm.js:1870
ee @ vue.runtime.esm.js:1839
t._render @ vue.runtime.esm.js:3550
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
(anonymous) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonymous) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (async)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
rr.set @ vue.runtime.esm.js:4631
(anonymous) @ group.vue:592
u @ runtime.js:45
(anonymous) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
Promise.then (async)
r @ asyncToGenerator.js:13
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
fetchGuild @ group.vue:585
(anonymous) @ group.vue:502
u @ runtime.js:45
(anonymous) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
mounted @ group.vue:499
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
insert @ vue.runtime.esm.js:3139
E @ vue.runtime.esm.js:6346
(anonymous) @ vue.runtime.esm.js:6565
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonymous) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (async)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
(anonymous) @ app.vue:400
Promise.then (async)
created @ app.vue:399
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
t._init @ vue.runtime.esm.js:5008
a @ vue.runtime.esm.js:5154
on @ vue.runtime.esm.js:3283
init @ vue.runtime.esm.js:3114
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
(anonymous) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3945
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
t._init @ vue.runtime.esm.js:5018
xr @ vue.runtime.esm.js:5085
56d7 @ main.js:36
c @ bootstrap:89
0 @ app.35ff403e.js:1
c @ bootstrap:89
a @ bootstrap:45
(anonymous) @ bootstrap:267
(anonymous) @ app.35ff403e.js:1
logging.js:33 Info: render
```

i also get js console errors, on firefox, chrome, and edge (all windows 10). i'm @dicedweller on habitica.

### Firefox:

Diese Seite verwendet die nicht standardisierte Eigenschaft "zoom". Stattdessen sollte calc() in den entsprechenden Eigenschaftswerten oder "transform" zusammen mit "transform-origin: 0 0" verwendet werden. party
Einige Cookies verwenden das empfohlene "SameSite"-Attribut inkorrekt. 8
Mutations-Ereignisse sollten nicht mehr verwendet werden. Verwenden Sie MutationObserver stattdessen. analyser.js:485:17
Feature Policy: Unbekannte Funktionalität (Feature) "payment" wird ignoriert. v3:1:38621
Feature Policy: Unbekannte Funktionalität (Feature) "payment" wird ignoriert. v3:1:38713
Unhandled error in Vue.js code. logging.js:30:12
    errorHandler logging.js:30
    VueJS 32
    e group.vue:592
    u runtime.js:45
    k runtime.js:271
    e runtime.js:97
    r Lodash
    s Babel
    (Async: promise callback)
    r chunk-vendors.6a592fd7.js:94
    Babel 3
    fetchGuild group.vue:585
    e group.vue:502
    u runtime.js:45
    k runtime.js:271
    e runtime.js:97
    r Lodash
    Babel 3
    mounted group.vue:499
    VueJS 18
    created app.vue:400
    (Async: promise callback)
    created app.vue:399
    VueJS 17
    56d7 main.js:36
    Webpack 6
Error: RangeError: invalid language tag: "en_GB"
    Bh localizeNumber.js:13
    ft questSidebarSection.vue:1
    VueJS 30
    e group.vue:592
    u runtime.js:45
    k runtime.js:271
    e runtime.js:97
    r Lodash
    s Babel
    promise callback*r chunk-vendors.6a592fd7.js:94
    Babel 3
    fetchGuild group.vue:585
    e group.vue:502
    u runtime.js:45
    k runtime.js:271
    e runtime.js:97
    r Lodash
    Babel 3
    mounted group.vue:499
    VueJS 18
    created app.vue:400
    promise callback*created app.vue:399
    VueJS 17
    56d7 main.js:36
    Webpack 6
logging.js:31:12
    errorHandler logging.js:31
    VueJS 32
    e group.vue:592
    u runtime.js:45
    k runtime.js:271
    e runtime.js:97
    r Lodash
    s Babel
    (Async: promise callback)
    r chunk-vendors.6a592fd7.js:94
    Babel 3
    fetchGuild group.vue:585
    e group.vue:502
    u runtime.js:45
    k runtime.js:271
    e runtime.js:97
    r Lodash
    Babel 3
    mounted group.vue:499
    VueJS 18
    created app.vue:400
    (Async: promise callback)
    created app.vue:399
    VueJS 17
    56d7 main.js:36
    Webpack 6
Component where it occurred: 
Object { _uid: 1018, _isVue: true, "$options": {…}, _renderProxy: {…}, _self: {…}, "$parent": {…}, "$root": {…}, "$children": [], "$refs": {}, _watcher: {…}, … }
logging.js:32:12
    errorHandler logging.js:32
    VueJS 32
    e group.vue:592
    u runtime.js:45
    k runtime.js:271
    e runtime.js:97
    r Lodash
    s Babel
    (Async: promise callback)
    r chunk-vendors.6a592fd7.js:94
    Babel 3
    fetchGuild group.vue:585
    e group.vue:502
    u runtime.js:45
    k runtime.js:271
    e runtime.js:97
    r Lodash
    Babel 3
    mounted group.vue:499
    VueJS 18
    created app.vue:400
    (Async: promise callback)
    created app.vue:399
    VueJS 17
    56d7 main.js:36
    Webpack 6
Info: render logging.js:33:12
    errorHandler logging.js:33
    VueJS 32
    e group.vue:592
    u runtime.js:45
    k runtime.js:271
    e runtime.js:97
    r Lodash
    s Babel
    (Async: promise callback)
    r chunk-vendors.6a592fd7.js:94
    Babel 3
    fetchGuild group.vue:585
    e group.vue:502
    u runtime.js:45
    k runtime.js:271
    e runtime.js:97
    r Lodash
    Babel 3
    mounted group.vue:499
    VueJS 18
    created app.vue:400
    (Async: promise callback)
    created app.vue:399
    VueJS 17
    56d7 main.js:36
    Webpack 6
Feature Policy: Unbekannte Funktionalität (Feature) "payment" wird ignoriert. v3:1:37908
Error: Could not establish connection. Receiving end does not exist. mcafee_wa_content.js:23:63330
Error: Could not establish connection. Receiving end does not exist. mcafee_wa_content.js:23:12894
Source-Map-Fehler: Error: NetworkError when attempting to fetch resource.
Ressourcen-Adresse: moz-extension://f9b7afce-c0fa-43a6-b6e9-34ed15868513/mcafee_wa_content.js
Source-Map-Adresse: ../sourceMap/firefox/mcafee_wa_content.map

​

### Chrome (with dark mode extension):

Unhandled error in Vue.js code.
n.default.config.errorHandler @ logging.js:30
re @ vue.runtime.esm.js:1870
ee @ vue.runtime.esm.js:1839
t._render @ vue.runtime.esm.js:3550
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
(anonymous) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonymous) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (async)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
rr.set @ vue.runtime.esm.js:4631
(anonymous) @ group.vue:592
u @ runtime.js:45
(anonymous) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
Promise.then (async)
r @ asyncToGenerator.js:13
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
fetchGuild @ group.vue:585
(anonymous) @ group.vue:502
u @ runtime.js:45
(anonymous) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
mounted @ group.vue:499
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
insert @ vue.runtime.esm.js:3139
E @ vue.runtime.esm.js:6346
(anonymous) @ vue.runtime.esm.js:6565
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonymous) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (async)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
(anonymous) @ app.vue:400
Promise.then (async)
created @ app.vue:399
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
t._init @ vue.runtime.esm.js:5008
a @ vue.runtime.esm.js:5154
on @ vue.runtime.esm.js:3283
init @ vue.runtime.esm.js:3114
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
(anonymous) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3945
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
t._init @ vue.runtime.esm.js:5018
xr @ vue.runtime.esm.js:5085
56d7 @ main.js:36
c @ bootstrap:89
0 @ app.35ff403e.js:1
c @ bootstrap:89
a @ bootstrap:45
(anonymous) @ bootstrap:267
(anonymous) @ app.35ff403e.js:1
logging.js:31 Error: RangeError: Incorrect locale information provided
    at Number.toLocaleString (<anonymous>)
    at Bh (localizeNumber.js:13)
    at a.ft (questSidebarSection.vue?c281:1)
    at a.t._render (vue.runtime.esm.js:3548)
    at a.r (vue.runtime.esm.js:4066)
    at nr.get (vue.runtime.esm.js:4479)
    at new nr (vue.runtime.esm.js:4468)
    at $n (vue.runtime.esm.js:4073)
    at a.xr.$mount (vue.runtime.esm.js:8415)
    at init (vue.runtime.esm.js:3118)
n.default.config.errorHandler @ logging.js:31
re @ vue.runtime.esm.js:1870
ee @ vue.runtime.esm.js:1839
t._render @ vue.runtime.esm.js:3550
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
(anonymous) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonymous) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (async)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
rr.set @ vue.runtime.esm.js:4631
(anonymous) @ group.vue:592
u @ runtime.js:45
(anonymous) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
Promise.then (async)
r @ asyncToGenerator.js:13
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
fetchGuild @ group.vue:585
(anonymous) @ group.vue:502
u @ runtime.js:45
(anonymous) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
mounted @ group.vue:499
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
insert @ vue.runtime.esm.js:3139
E @ vue.runtime.esm.js:6346
(anonymous) @ vue.runtime.esm.js:6565
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonymous) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (async)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
(anonymous) @ app.vue:400
Promise.then (async)
created @ app.vue:399
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
t._init @ vue.runtime.esm.js:5008
a @ vue.runtime.esm.js:5154
on @ vue.runtime.esm.js:3283
init @ vue.runtime.esm.js:3114
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
(anonymous) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3945
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
t._init @ vue.runtime.esm.js:5018
xr @ vue.runtime.esm.js:5085
56d7 @ main.js:36
c @ bootstrap:89
0 @ app.35ff403e.js:1
c @ bootstrap:89
a @ bootstrap:45
(anonymous) @ bootstrap:267
(anonymous) @ app.35ff403e.js:1
logging.js:32 Component where it occurred: a {_uid: 1018, _isVue: true, $options: {…}, _renderProxy: a, _self: a, …}$attrs: (...)$children: []$createElement: ƒ (e,n,r,i)$el: comment$listeners: (...)$options: {parent: a, _parentVnode: yt, propsData: {…}, _parentListeners: undefined, _renderChildren: undefined, …}$parent: a {_uid: 156, _isVue: true, $options: {…}, _renderProxy: a, _self: a, …}$refs: {}$root: xr {_uid: 3, _isVue: true, $options: {…}, _renderProxy: xr, _self: xr, …}$scopedSlots: {$stable: true, $key: undefined, $hasNormal: false}$slots: {}$store: e {_actions: {…}, getters: {…}, _vm: xr}$vnode: yt {tag: "vue-component-300-quest-sidebar-section", data: {…}, children: undefined, text: undefined, elm: comment, …}acceptedCount: (...)bossHpPercent: (...)canEditQuest: (...)group: (...)icons: (...)isMemberOfPendingQuest: (...)isMemberOfRunningQuest: (...)onActiveQuest: (...)onPendingQuest: (...)openParticipantList: ƒ ()openQuestDetails: ƒ ()openStartQuestModal: ƒ ()questAbort: ƒ ()questAccept: ƒ ()questData: (...)questLeave: ƒ ()questReject: ƒ ()user: (...)userIsOnQuest: (...)_bv__modal: t {_vm: a, _root: xr}_bv__toast: t {_vm: a, _root: xr}_c: ƒ (e,n,r,i)_computedWatchers: {user: nr, userIsOnQuest: nr, onPendingQuest: nr, onActiveQuest: nr, bossHpPercent: nr, …}_data: {__ob__: Et}_directInactive: false_events: {}_hasHookEvent: false_inactive: null_isBeingDestroyed: false_isDestroyed: false_isMounted: true_isVue: true_props: {}_renderProxy: a {_uid: 1018, _isVue: true, $options: {…}, _renderProxy: a, _self: a, …}_routerRoot: xr {_uid: 3, _isVue: true, $options: {…}, _renderProxy: xr, _self: xr, …}_self: a {_uid: 1018, _isVue: true, $options: {…}, _renderProxy: a, _self: a, …}_staticTrees: (5) [empty × 3, yt, yt]_uid: 1018_vnode: yt {tag: undefined, data: undefined, children: undefined, text: "", elm: comment, …}_watcher: nr {vm: a, deep: false, user: false, lazy: false, sync: false, …}_watchers: (11) [nr, nr, nr, nr, nr, nr, nr, nr, nr, nr, nr]$bvModal: (...)$bvToast: (...)$data: (...)$isServer: (...)$props: (...)$route: (...)$router: (...)$ssrContext: (...)get $attrs: ƒ ()set $attrs: ƒ (e)get $listeners: ƒ ()set $listeners: ƒ (e)get icons: ƒ ()set icons: ƒ (t)__proto__: xr
n.default.config.errorHandler @ logging.js:32
re @ vue.runtime.esm.js:1870
ee @ vue.runtime.esm.js:1839
t._render @ vue.runtime.esm.js:3550
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
(anonymous) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonymous) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (async)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
rr.set @ vue.runtime.esm.js:4631
(anonymous) @ group.vue:592
u @ runtime.js:45
(anonymous) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
Promise.then (async)
r @ asyncToGenerator.js:13
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
fetchGuild @ group.vue:585
(anonymous) @ group.vue:502
u @ runtime.js:45
(anonymous) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
mounted @ group.vue:499
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
insert @ vue.runtime.esm.js:3139
E @ vue.runtime.esm.js:6346
(anonymous) @ vue.runtime.esm.js:6565
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonymous) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (async)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
(anonymous) @ app.vue:400
Promise.then (async)
created @ app.vue:399
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
t._init @ vue.runtime.esm.js:5008
a @ vue.runtime.esm.js:5154
on @ vue.runtime.esm.js:3283
init @ vue.runtime.esm.js:3114
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
(anonymous) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3945
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
t._init @ vue.runtime.esm.js:5018
xr @ vue.runtime.esm.js:5085
56d7 @ main.js:36
c @ bootstrap:89
0 @ app.35ff403e.js:1
c @ bootstrap:89
a @ bootstrap:45
(anonymous) @ bootstrap:267
(anonymous) @ app.35ff403e.js:1
logging.js:33 Info: render



### Edge:

Unhandled error in Vue.js code.
n.default.config.errorHandler @ logging.js:30
re @ vue.runtime.esm.js:1870
ee @ vue.runtime.esm.js:1839
t._render @ vue.runtime.esm.js:3550
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
(anonym) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonym) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (asynchron)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
rr.set @ vue.runtime.esm.js:4631
(anonym) @ group.vue:592
u @ runtime.js:45
(anonym) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
Promise.then (asynchron)
r @ asyncToGenerator.js:13
s @ asyncToGenerator.js:25
(anonym) @ asyncToGenerator.js:32
(anonym) @ asyncToGenerator.js:21
fetchGuild @ group.vue:585
(anonym) @ group.vue:502
u @ runtime.js:45
(anonym) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonym) @ asyncToGenerator.js:32
(anonym) @ asyncToGenerator.js:21
mounted @ group.vue:499
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
insert @ vue.runtime.esm.js:3139
E @ vue.runtime.esm.js:6346
(anonym) @ vue.runtime.esm.js:6565
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonym) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (asynchron)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
(anonym) @ app.vue:400
Promise.then (asynchron)
created @ app.vue:399
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
t._init @ vue.runtime.esm.js:5008
a @ vue.runtime.esm.js:5154
on @ vue.runtime.esm.js:3283
init @ vue.runtime.esm.js:3114
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
(anonym) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3945
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
t._init @ vue.runtime.esm.js:5018
xr @ vue.runtime.esm.js:5085
56d7 @ main.js:36
c @ bootstrap:89
0 @ app.35ff403e.js:1
c @ bootstrap:89
a @ bootstrap:45
(anonym) @ bootstrap:267
(anonym) @ app.35ff403e.js:1
logging.js:31 Error: RangeError: Incorrect locale information provided
    at Number.toLocaleString (<anonymous>)
    at Bh (localizeNumber.js:13)
    at a.ft (questSidebarSection.vue?c281:1)
    at a.t._render (vue.runtime.esm.js:3548)
    at a.r (vue.runtime.esm.js:4066)
    at nr.get (vue.runtime.esm.js:4479)
    at new nr (vue.runtime.esm.js:4468)
    at $n (vue.runtime.esm.js:4073)
    at a.xr.$mount (vue.runtime.esm.js:8415)
    at init (vue.runtime.esm.js:3118)
n.default.config.errorHandler @ logging.js:31
re @ vue.runtime.esm.js:1870
ee @ vue.runtime.esm.js:1839
t._render @ vue.runtime.esm.js:3550
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
(anonym) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonym) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (asynchron)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
rr.set @ vue.runtime.esm.js:4631
(anonym) @ group.vue:592
u @ runtime.js:45
(anonym) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
Promise.then (asynchron)
r @ asyncToGenerator.js:13
s @ asyncToGenerator.js:25
(anonym) @ asyncToGenerator.js:32
(anonym) @ asyncToGenerator.js:21
fetchGuild @ group.vue:585
(anonym) @ group.vue:502
u @ runtime.js:45
(anonym) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonym) @ asyncToGenerator.js:32
(anonym) @ asyncToGenerator.js:21
mounted @ group.vue:499
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
insert @ vue.runtime.esm.js:3139
E @ vue.runtime.esm.js:6346
(anonym) @ vue.runtime.esm.js:6565
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonym) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (asynchron)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
(anonym) @ app.vue:400
Promise.then (asynchron)
created @ app.vue:399
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
t._init @ vue.runtime.esm.js:5008
a @ vue.runtime.esm.js:5154
on @ vue.runtime.esm.js:3283
init @ vue.runtime.esm.js:3114
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
(anonym) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3945
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
t._init @ vue.runtime.esm.js:5018
xr @ vue.runtime.esm.js:5085
56d7 @ main.js:36
c @ bootstrap:89
0 @ app.35ff403e.js:1
c @ bootstrap:89
a @ bootstrap:45
(anonym) @ bootstrap:267
(anonym) @ app.35ff403e.js:1
logging.js:32 Component where it occurred: a {_uid: 1018, _isVue: true, $options: {…}, _renderProxy: a, _self: a, …}$attrs: (...)$children: []$createElement: ƒ (e,n,r,i)$el: comment$listeners: (...)$options: {parent: a, _parentVnode: yt, propsData: {…}, _parentListeners: undefined, _renderChildren: undefined, …}$parent: a {_uid: 156, _isVue: true, $options: {…}, _renderProxy: a, _self: a, …}$refs: {}$root: xr {_uid: 3, _isVue: true, $options: {…}, _renderProxy: xr, _self: xr, …}$scopedSlots: {$stable: true, $key: undefined, $hasNormal: false}$slots: {}$store: e {_actions: {…}, getters: {…}, _vm: xr}$vnode: yt {tag: "vue-component-300-quest-sidebar-section", data: {…}, children: undefined, text: undefined, elm: comment, …}acceptedCount: (...)bossHpPercent: (...)canEditQuest: (...)group: (...)icons: (...)isMemberOfPendingQuest: (...)isMemberOfRunningQuest: (...)onActiveQuest: (...)onPendingQuest: (...)openParticipantList: ƒ ()openQuestDetails: ƒ ()openStartQuestModal: ƒ ()questAbort: ƒ ()questAccept: ƒ ()questData: (...)questLeave: ƒ ()questReject: ƒ ()user: (...)userIsOnQuest: (...)_bv__modal: t {_vm: a, _root: xr}_bv__toast: t {_vm: a, _root: xr}_c: ƒ (e,n,r,i)_computedWatchers: {user: nr, userIsOnQuest: nr, onPendingQuest: nr, onActiveQuest: nr, bossHpPercent: nr, …}_data: {__ob__: Et}_directInactive: false_events: {}_hasHookEvent: false_inactive: null_isBeingDestroyed: false_isDestroyed: false_isMounted: true_isVue: true_props: {}_renderProxy: a {_uid: 1018, _isVue: true, $options: {…}, _renderProxy: a, _self: a, …}_routerRoot: xr {_uid: 3, _isVue: true, $options: {…}, _renderProxy: xr, _self: xr, …}_self: a {_uid: 1018, _isVue: true, $options: {…}, _renderProxy: a, _self: a, …}_staticTrees: (5) [leer x 3, yt, yt]_uid: 1018_vnode: yt {tag: undefined, data: undefined, children: undefined, text: "", elm: comment, …}_watcher: nr {vm: a, deep: false, user: false, lazy: false, sync: false, …}_watchers: (11) [nr, nr, nr, nr, nr, nr, nr, nr, nr, nr, nr]$bvModal: (...)$bvToast: (...)$data: (...)$isServer: (...)$props: (...)$route: (...)$router: (...)$ssrContext: (...)get $attrs: ƒ ()set $attrs: ƒ (e)get $listeners: ƒ ()set $listeners: ƒ (e)get icons: ƒ ()set icons: ƒ (t)__proto__: xr
n.default.config.errorHandler @ logging.js:32
re @ vue.runtime.esm.js:1870
ee @ vue.runtime.esm.js:1839
t._render @ vue.runtime.esm.js:3550
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
y @ vue.runtime.esm.js:6053
h @ vue.runtime.esm.js:5954
(anonym) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonym) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (asynchron)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
rr.set @ vue.runtime.esm.js:4631
(anonym) @ group.vue:592
u @ runtime.js:45
(anonym) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
Promise.then (asynchron)
r @ asyncToGenerator.js:13
s @ asyncToGenerator.js:25
(anonym) @ asyncToGenerator.js:32
(anonym) @ asyncToGenerator.js:21
fetchGuild @ group.vue:585
(anonym) @ group.vue:502
u @ runtime.js:45
(anonym) @ runtime.js:271
O.forEach.t.<computed> @ runtime.js:97
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:25
(anonym) @ asyncToGenerator.js:32
(anonym) @ asyncToGenerator.js:21
mounted @ group.vue:499
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
insert @ vue.runtime.esm.js:3139
E @ vue.runtime.esm.js:6346
(anonym) @ vue.runtime.esm.js:6565
In.t._update @ vue.runtime.esm.js:3948
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr.run @ vue.runtime.esm.js:4554
Zn @ vue.runtime.esm.js:4310
(anonym) @ vue.runtime.esm.js:1980
ue @ vue.runtime.esm.js:1906
Promise.then (asynchron)
oe @ vue.runtime.esm.js:1933
pe @ vue.runtime.esm.js:1990
tr @ vue.runtime.esm.js:4402
nr.update @ vue.runtime.esm.js:4544
mt.notify @ vue.runtime.esm.js:730
set @ vue.runtime.esm.js:1055
(anonym) @ app.vue:400
Promise.then (asynchron)
created @ app.vue:399
ne @ vue.runtime.esm.js:1854
Ln @ vue.runtime.esm.js:4219
t._init @ vue.runtime.esm.js:5008
a @ vue.runtime.esm.js:5154
on @ vue.runtime.esm.js:3283
init @ vue.runtime.esm.js:3114
p @ vue.runtime.esm.js:5978
h @ vue.runtime.esm.js:5925
(anonym) @ vue.runtime.esm.js:6516
In.t._update @ vue.runtime.esm.js:3945
r @ vue.runtime.esm.js:4066
nr.get @ vue.runtime.esm.js:4479
nr @ vue.runtime.esm.js:4468
$n @ vue.runtime.esm.js:4073
xr.$mount @ vue.runtime.esm.js:8415
t._init @ vue.runtime.esm.js:5018
xr @ vue.runtime.esm.js:5085
56d7 @ main.js:36
c @ bootstrap:89
0 @ app.35ff403e.js:1
c @ bootstrap:89
a @ bootstrap:45
(anonym) @ bootstrap:267
(anonym) @ app.35ff403e.js:1
logging.js:33

--- a/website/client/src/components/groups/questSidebarSection.vue
+++ b/website/client/src/components/groups/questSidebarSection.vue
@@ -198,7 +198,13 @@
               <div class="col-6">
                 <span
                   class="float-left"
-                >{{ $t('rage') }} {{ parseFloat(group.quest.progress.rage).toFixed(2) }} / {{ questData.boss.rage.value }}</span> <!-- eslint-disable-line max-len -->
+                >{{ $t('rage') }} {{
+                  parseFloat(group.quest.progress.rage)
+                    | localizeNumber(user.preferences.language, { toFixed:2 })
+                }} / {{
+                  questData.boss.rage.value
+                    | localizeNumber(user.preferences.language)
+                }}</span>
               </div>
             </div>
           </div>

--- a/website/client/src/filters/localizeNumber.js
+++ b/website/client/src/filters/localizeNumber.js
@@ -10,5 +10,13 @@ export default function localizeNumber (valIn, lang, optIn) {
     optOut.minimumFractionDigits = toFixed;
   }
 
-  return val.toLocaleString(lang, optOut);
+  return val.toLocaleString(
+    // Catch null just incase
+    (lang || '')
+      // Strip beyond @ symbol to allow custom languages
+      .replace(/@(?:.+)$/, '')
+      // We use underscore, this mthd uses dash
+      .replace('_', '-'),
+    optOut,
+  );
 }

--- a/website/client/src/filters/localizeNumber.js
+++ b/website/client/src/filters/localizeNumber.js
@@ -12,7 +12,7 @@ export default function localizeNumber (valIn, lang, optIn) {
 
   return val.toLocaleString(
     // Catch null just incase
-    (lang || '')
+    (lang || [])
       // Strip beyond @ symbol to allow custom languages
       .replace(/@(?:.+)$/, '')
       // We use underscore, this mthd uses dash

